browser/resources/settings/brave_overrides/autofill_page.ts

```diff
@@ -37,24 +60,21 @@ RegisterPolymerTemplateModifications({
           hidden=[[!isAutofillPage_]]
         </settings-toggle-button>
       `)
-      if (isEmailAliasesEnabled) {
-        const pages = templateContent.getElementById('pages')
-        if (!pages) {
-          console.error('[overrides]: Couldn\'t find pages')
-        } else {
-          pages.appendChild(html`
-          <template is="dom-if" route-path="/email-aliases">
-            <settings-subpage
-              associated-control="[[$$('#paymentManagerButton')]]"
-              page-title="${loadTimeData.getString('emailAliasesLabel')}"
-              learn-more-url="${loadTimeData.getString(
-                'addressesAndPaymentMethodsLearnMoreURL')}">
-              <settings-email-aliases-page id="emailAliasesSection"
-                prefs="{{prefs}}" />
-            </settings-subpage>
-          </template>`)
-        }
-      }
-    }
   },
-)
+  'settings-autofill-page-index': (templateContent) => {
+    if (!loadTimeData.getBoolean('isEmailAliasesEnabled')) {
+      return
+    }
+    const viewManager = templateContent.querySelector('cr-view-manager')
+    if (!viewManager) {
+      throw new Error(
+        '[Settings] Unable to find cr-view-manager on autofill-page')
+    }
+    viewManager.append(html`
+      <settings-subpage page-title="Email Aliases" slot="view"
+         id="email-aliases" data-parent-view-id="parent" prefs="{{prefs}}">
```

Comment:
Hey, unfortunately its not safe to render nested elements in the cr-view-manager. It expects whatever is slotted to be an implementation of a page (even though it mostly works, it breaks things like internal nesting and `getAssociatedControlFor`).

Either add the wrapping subpage in `brave/browser/resources/settings/email_aliases_page/email_aliases_page.ts` or in `brave/browser/resources/settings/email_aliases_page/email_aliases.tsx`

Reply:
done